x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -610,7 +610,7 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
             })}
             helpText={i18n.translate('xpack.ml.dataframe.analytics.create.excludedFieldsHelpText', {
               defaultMessage:
-                'Optionally select fields to be excluded from analysis. All other supported fields will be included',
+                'Select fields to exclude from analysis. All other supported fields are included.',
```

Comment:
Is this sentence needed:  All other supported fields are included.

Reply:
I think it's best keeping the second sentence in for clarity. Otherwise a user could think they need to manually exclude unsupported fields.